drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/ValueListUiColumn.java

```diff
@@ -76,12 +76,12 @@ public ValueListUiColumn(final List<HeaderMetaData> headerMetaData,
     @Override
     public void doEdit(final GridCell<String> cell,
                        final GridBodyCellRenderContext context,
-                       final Callback<GridCellValue<String>> callback) {
+                       final Consumer<GridCellValue<String>> callback) {
```

Comment:
Noticed in all files, we now have variables `Consumer callback`, should we refactor to `Consumer consumer`? Not sure, just asking.

Reply:
See comment above